emulators/hyperkit/Portfile

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+categories          emulators
+platforms           darwin
+maintainers         nomaintainer
+license             BSD
+
+github.setup        moby hyperkit 0.20210107 v
+github.tarball_from archive
+
+description         HyperKit is a toolkit for embedding hypervisor capabilities in your application.
+
+long_description    HyperKit is a toolkit for embedding hypervisor capabilities in your application. \
+                    It includes a complete hypervisor, based on xhyve/bhyve, which is optimized for \
+                    lightweight virtual machines and container deployment. It is designed to be interfaced \
+                    with higher-level components such as the VPNKit and DataKit.
+
+checksums           rmd160  49a397d86d20a7ea5d09261f3217b634d2a8b712 \
+                    sha256  095f5f5ef550d7cad10e4d13e9c9ce8b58cc319d654a6d837d8d87ee70537835 \
+                    size    1176356
+
+if {${os.platform} ne "darwin" || ${os.major} < 14} {
+    known_fail  yes
+
+    depebds_build
+    depends_lib
+
+    pre-fetch {
+        ui_error "${name} requires Hypervisor.framework from OS X Yosemite or later"
+        return -code error "incompatible OS X version"
+    }
+}
+
+depends_build       port:aspcud \
+                    port:ocaml \
+                    port:opam \
+                    port:pkgconfig
+
+depends_lib         port:libev
+
+                    # ref: https://github.com/moby/hyperkit/issues/323
+patchfiles          implicit-conversion.patch
+
+use_configure no
```

Comment:
Using `use_configure no` is an immediate red flag that your portfile may not be [using the right compiler and flags](https://trac.macports.org/wiki/UsingTheRightCompiler).